frontend/tests/controllers/QualityNominationTest.php

```diff
@@ -808,6 +808,108 @@ public function testDemotionCanBeBannedByReviewer() {
         );
     }
 
+    /**
+     * Check that can search nominations.
+     */
+    public function testSearchNominations() {
+        $problemsData = [\OmegaUp\Test\Factories\Problem::createProblem(),
+        \OmegaUp\Test\Factories\Problem::createProblem(),
+        \OmegaUp\Test\Factories\Problem::createProblem()];
+
+        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser();
+        $login = self::login($identity);
+
+        foreach ($problemsData as $problemData) {
+            \OmegaUp\Controllers\QualityNomination::apiCreate(new \OmegaUp\Request([
+                'auth_token' => $login->auth_token,
+                'problem_alias' => $problemData['request']['problem_alias'],
+                'nomination' => 'demotion',
+                'contents' => json_encode([
+                     'statements' => [
+                        'es' => [
+                            'markdown' => 'a + b',
+                        ],
+                     ],
+                     'rationale' => 'qwert',
+                     'reason' => 'offensive',
+                ]),
+            ]));
+            $alias[] = $problemData['request']['problem_alias'];
+            $authors[] = $problemData['request']['author_username'];
+        }
+
+        $reviewerLogin = self::login(
+            \OmegaUp\Test\Factories\QualityNomination::$reviewers[0]
+        );
+        $response = \OmegaUp\Controllers\QualityNomination::apiList(
+            new \OmegaUp\Request([
+                'auth_token' => $reviewerLogin->auth_token,
+            ])
+        );
+
+        $this->assertEquals(3, count($response['nominations']));
```

Comment:
```suggestion
        $this->assertCount(3, $response['nominations']);
```

`$this->assertCount(x, y)` se prefiere siempre a `$this->assertEquals(x, count(y))`

Reply:
listo